packages/angular_devkit/build_optimizer/src/transforms/wrap-enums.ts

```diff
@@ -276,7 +281,7 @@ function updateHostNode(
 }
 
 /**
- * Find enums, class expression or declaration statements.
+ * Find enums, class expression or declaration statemen
```

Comment:
typo?

Reply:
Yeah, looks like I removed it by mistake.